### PR TITLE
Updated PG upgrade steps with analyze commands

### PR DESCRIPTION
**Pull request by jashaik**
Signed-off-by: jan shahid shaik <jashaik@progress.com>

### Description

Updated chef server upgrade steps with PG ANALYZE commands 

### Issues Resolved
Identified while upgrading PG 9.6 to 13.4 in hosted chef environment.

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: docs-chef-io/content/server/upgrades.md [hunk 1]
```diff
@@ -76,6 +76,23 @@ Set the `postgresql['pg_upgrade_timeout']` attribute in [chef-server.rb]({{< rel
...
+
+   You should then see output like:
+   
+    ```bash
+       vacuumdb: vacuuming database "bifrost"
+       vacuumdb: vacuuming database "oc_id"
+       vacuumdb: vacuuming database "opscode-pgsql"
+       vacuumdb: vacuuming database "opscode_chef"
+       vacuumdb: vacuuming database "postgres"
+       vacuumdb: vacuuming database "template1"
```
Comment (PrajaktaPurohit) [inline docs-chef-io/content/server/upgrades.md:93]: Interesting - the changing flags do not seem to impact the output? It still looks like the db is being vacuumed.

File: docs-chef-io/content/server/upgrades.md [hunk 2]
```diff
@@ -218,6 +235,23 @@ If you are running a Chef Infra Server release before 12.3.0, please contact Che
...
```
Comment (PrajaktaPurohit) [inline docs-chef-io/content/server/upgrades.md:252]: We have suggested the vacuum full at the beginning of the process, but depending on the size of the data compared to the system that can also be done just before running analyze, optionally.

File: docs-chef-io/content/server/upgrades.md [hunk 3]
```diff
@@ -471,6 +505,22 @@ The following External PostgreSQL upgrade steps are provided as a courtesy only.
...
+    ```
```
Comment (PrajaktaPurohit) [inline docs-chef-io/content/server/upgrades.md:522]: Is there a section that links the commands to vacuum before this analyze? (I think this is the external PG section)

File: docs-chef-io/content/server/upgrades.md (hunk 1 above)
Comment (jashaik) [reply docs-chef-io/content/server/upgrades.md:93]: `/opt/opscode/embedded/bin/vacuumdb` is tool which accepts options like `--analyze-only` or `--full`. So the output is same if we use vacuumdb script.Both commands can be run on the psql console but needs to give the database names.

File: docs-chef-io/content/server/upgrades.md (hunk 3 above)
Comment (jashaik) [reply docs-chef-io/content/server/upgrades.md:522]: You mean just a link to Vacuum command ?

File: docs-chef-io/content/server/upgrades.md (hunk 2 above)
Comment (PrajaktaPurohit) [reply docs-chef-io/content/server/upgrades.md:252]: yes, the suggestion here would be that vacuum at the start is recommended to make the process faster, but the customers can also do that after the upgrade.

File: docs-chef-io/content/server/upgrades.md (hunk 3 above)
Comment (PrajaktaPurohit) [reply docs-chef-io/content/server/upgrades.md:522]: yes somewhere wee should mention even for external postgres that vacuum should be performed before running analyze

File: docs-chef-io/content/server/upgrades.md (hunk 2 above)
Comment (vinay-satish) [reply docs-chef-io/content/server/upgrades.md:252]: Have added them

**Review by vinay-satish — APPROVED** [review]
LGTM

**Pull request merged**